about.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script
+      src="https://kit.fontawesome.com/cb6c5078b1.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  <link rel="stylesheet" href="page.css">
+  <title>About StartUp Lab</title>
+</head>
+<body>
+  <div class="top-bar-section">
+    <div class="top-bar-container">
+    <ul class="top-bar">
+      <li>
+        <i class="fab fa-facebook-f"></i>
+      </li>
+      <li>
+        <i class="fab fa-twitter"></i>
+      </li>
+      <li>
+        <h4>English</h4>
+      </li>
+      <li>
+        <h4>My Page</h4>
+      </li>
+      <li>
+        <h4>Logout</h4>
+      </li>
+    </ul>
+    </div>
```

Comment:
- [ ] Reduce the margins between these elements to resemble the original design

- [ ] Change the black background to the correct grey color

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/128281745-cacaf4f4-5160-47c8-a746-9e7758fb8991.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/128281860-fcdb5a63-e213-4167-b5b1-2deb34a64f9e.png)